deploy/tests/snapshots/generated_crds__generated_crds.snap

```diff
@@ -0,0 +1,192 @@
+---
```

Comment:
Is this filename as intended? `generated_crds__generated_crds.snap`

Reply:
Yes. `insta` first uses of the name of the Rust file where the test originated from and the name of the test itself. I'll change the name of that file so it's alittle more clear

Reply:
The name of this file should now be `insta_test__generated_crds.snap` which is the `<test-filename>__<test-name>.snap`